examples/practice-typescript/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,12 @@
+import './logo.scss';
+
+const Logo = () => {
+  return (
+    <div className="logo-group">
+      <a className="icon-logo" href="javascript:void(0)"></a>
+      <h1 className="title">Company name</h1>
```

Comment:
So do we have any behavior on this Company name? usually on logo we can click any where to navigate back to the homepage

Reply:
Fixed